docs/en/standalone.md

```diff
@@ -0,0 +1,134 @@
+# Standalone Mode Guideline
+
+## Deployment
+### Modify Config
+1. Replace 127.0.0.1 with the real IP if may be accessed in another node
```

Comment:
=>
1. Please replace `127.0.0.1` with the server IP if you want to access remotely.